lib/models/Player.js

```diff
@@ -283,6 +283,31 @@ function Player(data, listener, system) {
 
   }
 
+  function groupMuteHandler(uuid, mute) {
+    if (uuid !== _this.uuid) {
+      // This was not intended for us, skip it.
+      return;
+    }
+
+    let previousMute = _this.groupState.mute;
+    _this.groupState.mute = mute == '1';
+
+    _this.emit('group-mute', {
+      uuid: _this.uuid,
+      previousMute,
+      newMute: _this.groupState.mute,
+      roomName: _this.roomName
+    });
+
+    _this.system.emit('group-mute', {
+      uuid: _this.uuid,
+      previousMute,
+      newMute: _this.groupState.mute,
+      roomName: _this.roomName
+    });
+  }
+
+  listener.on('group-mute', groupMuteHandler);
   listener.on('last-change', notificationHandler);
 }
```

Comment:
In addition, if this is not emitted as a separate event (but instead as LastChange), it would already be handled by notificationHandler making this redundant.